src/framework/audio/internal/worker/mixerchannel.cpp

```diff
@@ -167,7 +167,7 @@ void MixerChannel::setSampleRate(unsigned int sampleRate)
         m_audioSource->setSampleRate(sampleRate);
     }
 
-    for (IFxProcessorPtr fx : m_fxProcessors) {
+    for (const IFxProcessorPtr& fx : m_fxProcessors) {
```

Comment:
It should not be const

Reply:
Only the _pointee_ is non-const, but the pointer itself can be const just fine (we're not going to replace it with a different pointer), right?

Reply:
It just looks a bit confusing imo: a non-constant reference would more clearly say that we're going to change the object